### PR TITLE
[Share] CAES-258: remastered endpoint for api/share(s)

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: deploy/deployment.yaml [hunk 1]
```diff
@@ -1,144 +1,156 @@
...
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /etc/nginx/nginx.conf
+          name: nginx-config-volume
+          subPath: nginx.conf
+        - mountPath: /var/www/html/public
+          name: caesar-shared-static
+      dnsPolicy: ClusterFirst
+      imagePullSecrets:
+      - name: dockerhub
```
Comment (beshkenadze) [inline deploy/deployment.yaml]: Why do we have so many duplicates here?

Comment (dyudin4xxi) [reply deploy/deployment.yaml]: @beshkenadze What are duplicates?

Comment (beshkenadze) [reply deploy/deployment.yaml]: @dyudin4xxi this one 
https://github.com/caesar-team/caesar-server/blob/6c7f83f27de9ea5f8a9ea00474e7b029b9603308/deploy/deployment.yaml#L124

File: src/Migrations/2019/02/Version20190211144134.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```
Comment (ipopov4xxi) [inline src/Migrations/2019/02/Version20190211144134.php]: Useless phpdoc

Comment (mg-4xxi) [reply src/Migrations/2019/02/Version20190211144134.php]: removed

File: deploy/deployment.yaml (hunk 1 above)
Comment (dyudin4xxi) [reply deploy/deployment.yaml]: @beshkenadze For each of the three images launched by the docker, access to the docker hub is prescribed; if one of them is removed, then everything will break

**Pull request merged**